### ISSUE TITLE
[Feature Request] img2img support for animatediff-cli

### ISSUE THREAD

**Issue by rbychn** [neggles/animatediff-cli#1]
Would be great to transform an already existing image and animate it instead of first generating using prompts and then animating.

Maybe also batch support so it can take multiple images from a folder and then transform all of them into gifs automatically without manual prompting.

**Comment by neggles** [comment]
True! This is very much on my list of things to do, once I sort out the attention code it should be fairly straightforward (I think... famous last words and all that) 

Assuming it works, it should be possible to chain one animation into another, too, so we won't be stuck at ~16 frames, which would be nice. Will update this when I have news :)

**Comment by rbychn** [comment]
@neggles Check this out for inspiration https://github.com/rynmurdock/inanimate

**Comment by toyxyz** [comment]
This guy's result looks amazing! https://github.com/ykk648/AnimateDiff

![image](https://github.com/neggles/animatediff-cli/assets/8006000/f2dbe63e-f552-4002-9588-80a6d73b7804)


![257777597-16c7079c-f6da-4316-aeca-e63b2c777529](https://github.com/neggles/animatediff-cli/assets/8006000/603f4e1f-9f4e-4f52-a8fe-76076665c785)


![257777301-b8ece59f-54b1-4bab-b18d-7d038be4dbc6](https://github.com/neggles/animatediff-cli/assets/8006000/0acce889-f101-4e04-abc2-3c012fbabed4)

**Comment by 15704080** [comment]
Looking forward to it very much

**Comment by rbychn** [comment]
while a lot of other things are fixed, this one is still pending, any ETA on when you'd be looking into this? :)

would be cool to get start and end frames (or multiple frames option) https://x.com/TDS_95514874/status/1693376863052390774
